bin/impl/fetch.sh

```diff
@@ -39,12 +31,15 @@ APACHE_MIRROR=$(curl -sk https://apache.org/mirrors.cgi?as_json | grep preferred
 
 case "$1" in
 hadoop)
+  [[ $HADOOP_VERSION =~ .*-incubating ]] && APACHE_MIRROR="${APACHE_MIRROR}/incubator"
```

Comment:
I don't think anyone will want to use incubating versions of Hadoop, Zookeeper, Spark, or Accumulo.  Can we remove these?  Makes sense to leave it for Fluo.